lib/recorder.js

```diff
@@ -235,26 +234,10 @@ function record(recOptions) {
   intercept.restoreOverriddenClientRequest()
 
   //  We override the requests so that we can save information on them before executing.
-  common.overrideRequests(function(
-    proto,
-    overriddenRequest,
-    options,
-    callback
-  ) {
+  common.overrideRequests(function(proto, overriddenRequest, rawArgs) {
+    const { options, callback } = common.normalizeClientRequestArgs(...rawArgs)
     const bodyChunks = []
 
-    if (typeof options === 'string') {
```

Comment:
No longer needed as we're explicitly normalizing the options?

Reply:
Correct, `options` is now guaranteed to always be an object by this point.